recipes/virulencefinder/download-virulence-db.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Thank you for the review and comments, and yes, that makes sense, it will make it easier for the user installment-wise. (And just to answer your question regarding the INSTALL.py script, this is run after the database has been downloaded for indexing the database, which is necessary if the user wants to use the program with KMA.)

The problem is, that we want to make it possible for the user to retrieve older versions of the database and still run it with the conda package of virulencefinder (for reproducibility). By installing the program and the database in one go, it would be difficult to reproduce for example results made earlier where the program and database has been installed separately. Every time the database is updated, we want to avoid updating the version of the program.  However, it should be noted that I'm fairly new to Bioconda, so if you see a good way of working around this, it would be greatly appreciated.

Reply:
So let me describe a bit what I meant, but I think you know your use case best, so just do it as you think best.

My argument goes that coupling the tool and the database would actually increase reproducibility because the user needs to keep track of one version number. That is, if you install version x of this tool now, you get exactly the same next year, both tool and accompanying database. The way it works now is that you get the most recent database version (when you run the `download-...sh` script) even if you install an older version of the Conda package. So somehow I as a user have to keep track of not only the version number of the tool, but I will also have to store the database that I downloaded somewhere (or in some other way remember how to get that old version).

But of course it’s not that hard to keep the downloaded database around, so you may come to a different conclusion. (You should consider adding version numbers to your database in case you are not doing that, yet.)

Reply:
Yes, that definitely makes sense!
I've spoken to some colleagues, and we agreed on having the datebase downloaded in a separate step. However, we also agreed on making in possible to specify which commit to download (default latest) in this script. I will make these changes. It is my first time contributing to Bioconda, so if I make these changes, what is the procedure then? Do I commit and then make a new pull request or how does it work?

Reply:
Just make further commits to your `update_virulencefinder` branch and then push to your repository. That will update this PR.

Reply:
Ah okay perfect, I'll do that! Thank you